src/main/java/com/xceptance/xlt/clientperformance/WebExtConnectionHandler.java

```diff
@@ -168,17 +168,25 @@ public void onConnect(ClientPerformanceExtensionConnector connector, ClientPerfo
     @Override
     public void onMessage(ClientPerformanceExtensionConnection connection, JSONObject data, Responder responder)
     {
+        final String action = data.optString("action");
+
+        LOG.debug("Message received: {}", action);
+
         if (LOG.isTraceEnabled())
         {
-            LOG.trace("Message received: " + data);
+            LOG.trace("Message data: " + data);
```

Comment:
One of the log statements seems superfluous to me. It should be sufficient to log the entire message data and remove the first log statement that logs the value of the action property only. WDYT?

Reply:
The message with all the request and event details might get big, for example half an MB. Printing this amount of data as a one-liner to the console slows down my Eclipse considerably and scrolling the console gets sluggish.

For this reason, I would like to log only the type of message just arrived and maybe how many entries I received if the message was a details message. If needed, the logging of the raw data can still be enabled.

Reply:
Well, we might think about to limit the output by adjusting `Message#toString()` (e.g. `StringUtils.abbreviate`) in any case.

We may keep the 1st log statement, but its message is little bit confusing as `action` is not the message received; `Message received for action '{}'` fits better IMO.

I also noticed that `ClientPerformanceExtensionConnector` already logs the deserliazed message at level TRACE. So, the entire if-statement is indeed superfluous.

Reply:
Since the whole message is indeed printed elsewhere, I will remove the logger call here and another one farther below.